app/models/region.rb

```diff
@@ -135,8 +135,9 @@ def syncable_patients
     case region_type
       when "block"
         registered_patients.with_discarded
-          .or(assigned_patients.with_discarded)
-          .union(appointed_patients.with_discarded)
+          .select(:id, :registration_facility_id, :updated_at)
+          .or(assigned_patients.with_discarded.select(:id, :registration_facility_id, :updated_at))
+          .union(appointed_patients.with_discarded.select(:id, :registration_facility_id, :updated_at))
```

Comment:
What's the use for selecting `registration_facility_id` and `updated_at` here? Why not just `select(:id)`?